src/mail-app/mail/model/MailModel.ts

```diff
@@ -51,6 +51,7 @@ import { LoginController } from "../../../common/api/main/LoginController.js"
 import { MailFacade } from "../../../common/api/worker/facades/lazy/MailFacade.js"
 import { assertSystemFolderOfType } from "./MailUtils.js"
 import { isSpamOrTrashFolder } from "./MailChecks.js"
+import { MailItem } from "../view/ConversationViewModel"
```

Comment:
`MailModel` should not depend on `ConversationViewModel`. I think we don't even need this import.